src/main/java/wolox/training/models/Book.java

```diff
@@ -62,6 +63,7 @@ public String getIsbn() {
     }
 
     public void setIsbn(String isbn) {
```

Comment:
Add the following validations:

- That the field is not empty

- That the field only contains numerical values